sql/migrations/20220107120001_world.sql

```diff
@@ -0,0 +1,380 @@
+DROP PROCEDURE IF EXISTS add_migration;
+delimiter ??
+CREATE PROCEDURE `add_migration`()
+BEGIN
+DECLARE v INT DEFAULT 1;
+SET v = (SELECT COUNT(*) FROM `migrations` WHERE `id`='20220107120001');
+IF v=0 THEN
+INSERT INTO `migrations` VALUES ('20220107120001');
+-- Add your query below.
+
+
+SET @PYREWOOD_EVENT = 49;
+SET @PYREWOOD_VILLAGE_CGUID = 57441;
+
+-- New Event for Pyrewood starts 21:00, ends 05:30 (confirmed by ptr).
+INSERT INTO `game_event` (`entry`, `start_time`, `end_time`, `occurence`, `length`, `holiday`, `description`, `hardcoded`, `disabled`, `patch_min`, `patch_max`) VALUES (@PYREWOOD_EVENT, '2020-01-02 21:00:00', '2030-12-31 05:30:00', 1440, 720, 0, 'Pyrewood Village Night', 0, 0, 0, 10);
```

Comment:
do you need to update length to correspond with 5:30am? Also should the silverpine script be updated to 5:30am for the pyrewood ambush transforms?

Reply:
i've already done everything incl. moved entire ambush quest to db. but not pushed yet.

Reply:
what about the length of 12 hours?